index.html

```diff
@@ -221,6 +221,22 @@ <h3 class="head3" id="social-2">LET’S CONNECT</h3>
         </div>
       </section>
     </main>
+    <footer class="footer" id="contacts">
+      <div class="content-placeholder" id="contact-placeholder">
+        <h2 class="head-1" id="contact-me">Contact Me</h2>
+        <p id="contact-msg">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>    
+        <form class="form" action="https://formspree.io/f/xbjwpzkw" method="post">
```

Comment:
- [ ] Kindly use Flexbox to layout the elements as per the project requirement. 
![contact-form-elem-flex-layout](https://user-images.githubusercontent.com/44057863/166561011-9a8b0c72-d3f5-4604-aa7c-9efc8ebc567f.png)